integration/testdata/helm-multi-config/skaffold/app2/skaffold.yml

```diff
@@ -15,5 +15,5 @@ manifests:
         createNamespace: true
         name: app2
         setValueTemplates:
-          image.repository: '{{.app2.IMAGE_REPO}}'
-          image.tag: "{{.app2.IMAGE_TAG}}@{{.app2.IMAGE_DIGEST}}"
\ No newline at end of file
+          image.repository: '{{.IMAGE_REPO_app2}}'
+          image.tag: "{{.IMAGE_TAG_app1}}@{{.IMAGE_DIGEST_app2}}"
```

Comment:
I think it should be `"{{.IMAGE_TAG_app2}}@{{.IMAGE_DIGEST_app2}}"` -> changed `app1` to `app2`

Reply:
Fixed thanks, I fixed that and then must have unfixed it same point lol